searchcore/src/vespa/searchcore/proton/server/documentdbconfigmanager.cpp

```diff
@@ -281,6 +282,9 @@ vespalib::string resolve_file(config::RpcFileAcquirer &fileAcquirer, vespalib::T
         }
     }
     LOG(info, "Got file path from file acquirer: '%s' (%s, ref='%s')", filePath.c_str(), desc.c_str(), fileref.c_str());
+    if (filePath == "") {
+        throw config::ConfigTimeoutException("could not get file path from file acquirer");
```

Comment:
I think you should add the same information as in the info line above to the exception message.